editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -638,15 +638,13 @@ export const previewStore: StateCreator<
     }
   },
 
-  // TODO: Fix this!
-  // requestedFiles: () => {
-  //   const { computePassport } = get();
-  //   const currentRequestedFiles =
-  //     computePassport().data?.[PASSPORT_REQUESTED_FILES_KEY];
-  //   const emptyFileList = { required: [], recommended: [], optional: [] };
+  requestedFiles: () => {
+    const { computePassport } = get();
+    const currentRequestedFiles = computePassport().data?._requestedFiles;
```

Comment:
Using `computePassport().data?.[PASSPORT_REQUESTED_FILES_KEY];` here leads to failing tests 🙃 

I guess it must be something to do with circular imports though there was no indication of that given via Jest output.

I'll added a comment / extra variable here so find and replace in future would still hopefully point at this hardcoded `_requiredFiles` value.

There might be an argument to place all `PASSPORT_*` consts in a single shared location if we hit this sort of curiosity again.